frontend/src/app/(core)/sponsors-and-partners/page.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+import { Button } from '@/components'
+import { Hero } from '@/sections/SponsorsAndPartners/Hero'
+import { PartnerCommunities } from '@/sections/SponsorsAndPartners/PartnerCommunities'
+import { IndividualSponsors } from '@/sections/SponsorsAndPartners/Sponsors/IndividualSponsors'
+import { OrganizationSponsors } from '@/sections/SponsorsAndPartners/Sponsors/OrganizationSponsors'
+import Link from 'next/link'
+
+interface Sponsor {
+  tier: string
+  sponsor_details: {
+    name: string
+    logo: string
+    description?: string
+  }
+}
+
+interface Partner {
+  name: string
+  logo: string
+  website: string
+  description: string
+}
+
+interface PageData {
+  sponsors: Sponsor[]
+  partners: Partner[]
+}
+
+const Page = async () => {
+  let data: PageData | null = null
+
+  try {
+    const response = await fetch(`${process.env.NEXT_PUBLIC_API_URL}/sponsors-and-partners/`, {
```

Comment:
Can we use `fetchData` util function ?

Reply:
There is an issue in fetching the data by using the Fetchdata function, therefore a normal fetch function is used as for now.